transform/publication/transform.py

```diff
@@ -0,0 +1,67 @@
+
+from bmeg.emitter import new_emitter
+from bmeg.ioutils import reader
+from bmeg.util.cli import default_argument_parser
+from bmeg.util.logging import default_logging
+from bmeg.vertex import Publication
+from bmeg.edge import *  # noqa
+
+import glob
+import logging
+import sys
+import ujson
+
+DEFAULT_DIRECTORY = 'publication'
+
+
+def transform(
+    emitter_name="json",
+    output_dir="outputs",
+    emitter_directory=DEFAULT_DIRECTORY,
+    edge_names="**/*.Edge.json*"
+):
+    batch_size = 1000
+    emitter = new_emitter(name=emitter_name, directory=emitter_directory, prefix='stub')
+
+    # get the edges
+    path = '{}/{}'.format(output_dir, edge_names)
+    files = [filename for filename in glob.iglob(path, recursive=True) if 'stub' not in filename]
```

Comment:
Seems like it would be more efficient to pass in the edge files we want to parse rather than go through all of them.  Could be an argument with defaults that can be overridden.